src/test/styles.test.js

```diff
@@ -91,7 +91,7 @@ describe('with styles', () => {
       ${rule1}
     `
     shallow(<Comp />)
-    expectCSSMatches('.sc-a {} .b { background-color: blue; } .b:hover { -webkit-text-decoration: underline; text-decoration: underline; }')
```

Comment:
Any ideas why this doesn't get prefixed anymore? /cc @thysultan

Reply:
Vendor prefixing `text-decoration` was a bug.

Reply:
Perfect, thank you for confirming.